app/lib/actions/pulp3/orchestration/orphan_cleanup/remove_orphans.rb

```diff
@@ -6,13 +6,14 @@ class RemoveOrphans < Pulp3::Abstract
           def plan(proxy)
             if proxy.pulp3_enabled?
               sequence do
-                plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanRepositoryVersions, proxy)
                 if proxy.pulp_mirror?
                   plan_action(Actions::Pulp3::OrphanCleanup::RemoveUnneededRepos, proxy)
-                  plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanDistributions, proxy)
                   plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanAlternateContentSources, proxy)
                   plan_action(Actions::Pulp3::OrphanCleanup::DeleteOrphanRemotes, proxy)
                 end
+                # Deleting repos causes orphaned distributions, so delete them before the distributions.
```

Comment:
Is this comment accurate?

Reply:
Yeah, I tested it out. Worth a second look though.

Reply:
I was referring to the remove unneeded repos task rather than the repo version cleanup task.

Reply:
Aah..The change vs the comment threw me off a little..